### PR TITLE
Create re-usable component to add toast notifications, adds license revocation cap experience.

### PR DESCRIPTION
**Pull request by adamstankiewicz**
### Toasts
* Adds `ToastsProvider` and `Toasts` component.
* Uses `addToast` to create new toast notifications where relevant for adding, reminding, and revoking subscriptions users.

### Revocation UX
* Updates the revoke modal messaging to introduce a warning alert to inform the admin how many revokes they have remaining.
* If the customer runs out of remaining revocations per their plan, hide the "Revoke" action for activated licenses and show a warning alert with a link to the Support page.

### Misc. cleanup
* Updates scss imports due to Paragon upgrade and makes indenting in scss files consistent (4 spaces -> 2 spaces)
* Removes the following unnecessary components: `H1`, `H2`, and `H3`.

### CODE REVIEW

File: src/components/LicenseRevokeModal/index.jsx [hunk 1]
```diff
@@ -115,16 +131,14 @@ class LicenseRevokeModal extends React.Component {
...
-          )}
+          ) : error[0]}
         />
       </div>
     );
   }
 
   renderTitle() {
-    return <small>Are you sure you want to revoke access?</small>;
+    return <small>Are you sure you want to revoke license?</small>;
```
Comment (iloveagent57) [inline src/components/LicenseRevokeModal/index.jsx:141]: Nit: "Are you sure you want to revoke _this_ license?"

File: src/components/LicenseRevokeModal/index.jsx [hunk 2]
```diff
@@ -79,16 +72,39 @@ class LicenseRevokeModal extends React.Component {
...
     return (
       <React.Fragment>
-        {submitFailed && this.renderErrorMessage()}
-        <div className="license-details mb-4">
+        <StatusAlert
+          alertType="warning"
+          message={
+            <p className="m-0">
+              You have already revoked {revocations.applied} licenses. You
+              have {revocations.remaining} left on your plan.
```
Comment (iloveagent57) [inline src/components/LicenseRevokeModal/index.jsx:88]: Nit: "You have N revocations left on your plan" (?)

**Review by iloveagent57 — APPROVED** [review]
Couple of nits about wording, but this LGTM.

File: src/components/LicenseRevokeModal/index.jsx (hunk 1 above)
Comment (adamstankiewicz) [reply src/components/LicenseRevokeModal/index.jsx:162]: I agree. I was just going off the copy provided in the wireframe 🤷‍♂️ I will add _this_.

File: src/components/LicenseRevokeModal/index.jsx (hunk 2 above)
Comment (adamstankiewicz) [reply src/components/LicenseRevokeModal/index.jsx]: image

The wireframe has "You have N more left on your plan." Looks like I missed the word "more" but I agree it feels like we need the word "revocation" in there as well.

**Review by chavesj — APPROVED** [review]
Tested it out locally, everything looks great. Love the reusable `ToastsProvider`. One tiny nit but LGTM 🚢

File: src/components/Sidebar/index.jsx [hunk 3]
```diff
@@ -168,7 +166,7 @@ Sidebar.defaultProps = {
   enableSubscriptionManagementScreen: false,
   enableSamlConfigurationScreen: false,
   enableAnalyticsScreen: false,
-  onWidthChange: () => {},
+  onWidthChange: () => { },
```
[the remark this answers is not in the document]
Comment (adamstankiewicz) [reply src/components/Sidebar/index.jsx:169]: Another one of those "auto-format on save" clean up misses :)

**Pull request merged**